plugins/logitech-bulkcontroller/data/com.logitech.vc.rallybar.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="firmware">
+  <id>com.logitech.vc.rallybar.firmware</id>
+  <name>Logitech Rally Bar Firmware</name>
```

Comment:
I don't think you need to include this file.

Reply:
Removed